aea/cli/generate_key.py

```diff
@@ -165,7 +166,9 @@ def _generate_multiple_keys(
             if password is not None
             else crypto.private_key
         )
-        key_pairs.append({ADDRESS: crypto.address, PRIVATE_KEY: priv_key})
+        key_pairs.append(
+            {ADDRESS: crypto.address, PRIVATE_KEY: priv_key, LEDGER: type_}
+        )
```

Comment:
This ledger specifier will be used by the deployment builder to write keys for the agents. The solana deployments failed because the deployment builder was writing solana keys as ethereum keys since there was no specifier in the keys.json file.